ui/page/wallets/wallet_page.go

```diff
@@ -637,6 +665,61 @@ func (pg *WalletPage) layoutWatchOnlyWallets(gtx layout.Context) D {
 	})
 }
 
+func (pg *WalletPage) badWalletsSection(gtx layout.Context) layout.Dimensions {
+	m20 := values.MarginPadding20
+	m10 := values.MarginPadding10
+
+	layoutBadWallet := func(gtx C, badWallet *badWalletListItem, lastItem bool) D {
+		return layout.Inset{Top: m10, Bottom: m10}.Layout(gtx, func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Flex{}.Layout(gtx,
+						layout.Rigid(pg.Theme.Body2(badWallet.Name).Layout),
+						layout.Flexed(1, func(gtx C) D {
+							return layout.E.Layout(gtx, func(gtx C) D {
+								return layout.Inset{Right: values.MarginPadding10}.Layout(gtx, badWallet.deleteBtn.Layout)
+							})
+						}),
+					)
+				}),
+				layout.Rigid(func(gtx C) D {
+					if lastItem {
+						return D{}
+					}
+					return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding38, Right: values.MarginPaddingMinus10}.Layout(gtx, func(gtx C) D {
+						return pg.Theme.Separator().Layout(gtx)
+					})
+				}),
+			)
+		})
+	}
+
+	card := pg.card
+	card.Color = pg.Theme.Color.Surface
```

Comment:
card color is `Surface` by default.

Reply:
I copied this from somewhere, but will just correct here since you say this is the default color.

Reply:
`pg.card.Color` is changed from the default value somewhere else in this file.